### DISCUSSION TITLE
FBX support with Godot 4.3

### DISCUSSION THREAD

**Discussion by HeavensSword** [V-Sekai/unidot_importer#45] [General]
Hello, 

I was curious if this project is still being maintained and, if so, are there any plans to support Godot 4.3 which will allow for importing FBX without the need for them to be converted into glTF?

**Comment by lyuma** [comment]
Definitely part of the plan! Latest git version of Unidot has support for native fbx import with ufbx.
It currently requires two godot PRs to work:
* https://github.com/godotengine/godot/pull/88824
* https://github.com/godotengine/godot/pull/89532

For now, I have guarded it behind a `return false` in asset_meta.gd but it is getting close.

**Reply by HeavensSword** [reply]
Awesome, looking forward to it! Thanks

**Reply by Namakamoto** [reply]
Thank you! Highly appreciated
